packages/scaffold-config/test/unit/detect.spec.ts

```diff
@@ -219,6 +219,34 @@ describe('detectFramework', () => {
     })
   })
 
+  it(`Svelte and Vite`, async () => {
+    const projectPath = await scaffoldMigrationProject('svelte-vite-unconfigured')
+
+    fakeDepsInNodeModules(projectPath, [
+      { dependency: 'svelte', version: '3.0.0' },
+      { dependency: 'vite', version: '3.0.0' },
+    ])
+
+    const actual = await detectFramework(projectPath)
+
+    expect(actual.framework?.type).to.eq('svelte')
+    expect(actual.bundler?.type).to.eq('vite')
+  })
+
+  it(`Svelte and Vite`, async () => {
+    const projectPath = await scaffoldMigrationProject('svelte-webpack-unconfigured')
+
+    fakeDepsInNodeModules(projectPath, [
+      { dependency: 'svelte', version: '3.0.0' },
+      { dependency: 'webpack', version: '5.0.0' },
```

Comment:
Do we need a webpack 4 vs 5 test?

Reply:
That's a good question. I'm not sure how testing different versions of the frameworks add value for the scaffolding functionality.

Reply:
We can add a test for both versions but I don't think it adds much value

Reply:
Maybe we should shift away from `alpha` to `experimental`, since the alpha tag doesn't really mean much (we aren't going to support Svelte Alpha -> Svelte Beta -> Svelte Stable). Would be a good thing to bring up next ct-product meeting

Reply:
> We can add a test for both versions but I don't think it adds much value

Sounds good, just curious since we already had a webpack 5-specific test in this file